modules/xmpp/xmpp.js

```diff
@@ -469,7 +473,8 @@ export default class XMPP extends Listenable {
      * @returns {Promise} Resolves with an instance of a strophe muc.
      */
     createRoom(roomName, options, onCreateResource) {
-        let roomjid = `${roomName}@${this.options.hosts.muc}/`;
+        let roomjid = `${roomName}@${options.customDomain
```

Comment:
is the `customDomain` option really needed? why don't just set different `options.hosts.muc` for the lobby instance? I would probably simplify this config option while at it from `options.hosts.muc` to `mucJid` or something

Reply:
That is actually not possible, without doing a rewrite of the multidomain module.
As the lobby room is a different component, once it is created the jid of the room is returned from the backend. I will do the description in the PR now, to explain it.